x-pack/plugins/fleet/server/services/epm/registry/registry_url.ts

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { appContextService } from '../..';
-
-// from https://github.com/elastic/package-registry#docker (maybe from OpenAPI one day)
-// the unused variables cause a TS warning about unused values
-// chose to comment them out vs @ts-ignore or @ts-expect-error on each line
-
-const PRODUCTION_REGISTRY_URL_CDN = 'https://epr.elastic.co';
-const STAGING_REGISTRY_URL_CDN = 'https://epr-staging.elastic.co';
-const SNAPSHOT_REGISTRY_URL_CDN = 'https://epr-snapshot.elastic.co';
-
-// const PRODUCTION_REGISTRY_URL_NO_CDN = 'https://epr.ea-web.elastic.dev';
-// const STAGING_REGISTRY_URL_NO_CDN = 'https://epr-staging.ea-web.elastic.dev';
-// const SNAPSHOT_REGISTRY_URL_NO_CDN = 'https://epr-snapshot.ea-web.elastic.dev';
-
-const getDefaultRegistryUrl = (): string => {
-  const branch = appContextService.getKibanaBranch();
-  const isProduction = appContextService.getIsProductionMode();
-  if (!isProduction || branch === 'main') {
-    return SNAPSHOT_REGISTRY_URL_CDN;
-  } else if (appContextService.getKibanaVersion().includes('-SNAPSHOT')) {
-    return STAGING_REGISTRY_URL_CDN;
-  } else {
-    return PRODUCTION_REGISTRY_URL_CDN;
-  }
-};
-
-export const getRegistryUrl = (): string => {
-  const customUrl = appContextService.getConfig()?.registryUrl;
```

Comment:
We should keep this, to take a custom url from config e.g. localhost for testing, or clients using a custom EPR.

Reply:
I had this doubt as well, we discussed with @kpollich and agreed to remove it. But if there are valid cases for keeping it (like testing), I'll put it back.

Reply:
I think this has many uses, not just for testing: https://www.elastic.co/guide/en/kibana/master/fleet-settings-kb.html#fleet-data-visualizer-settings

Reply:
My mistake on greenlighting the removal - I forgot this is where the `getConfig()` lookup for `xpack.fleet.registryUrl` was implemented. We do need that and it will probably make CI happy since the test suites can point at the Dockerized EPR instance again 😅